core-services/release-controller/_releases/release-konflux-ocp-4.19.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "4.19.0-0.konflux-nightly",
+  "to": "release",
```

Comment:
Do you want these to go into `release` or perhaps an intermediate place like `release-konflux`?  Just thinking out loud mostly...

Reply:
Updated